docker/chronicle.dockerfile

```
@@ -25,16 +51,25 @@ RUN if [ "${RELEASE}" = "yes" ]; then \
   fi;
 
 WORKDIR /
-FROM ubuntu:focal AS domain
+FROM ubuntu:focal AS domain-base
```

```suggestion
FROM ubuntu:jammy AS domain-base
```
That change got things working for me, working off this branch.